tests/TicTacToeTest.elm

```diff
@@ -1,19 +1,15 @@
 module TicTacToeTest exposing (..)
 
-import TicTacToe exposing (..)
-import ElmTest exposing (..)
+import Test exposing (..)
+import Expect
 
-tests =
-  suite "TicTacToe Test"
-    [ 
-      test 
-      "" 
-      <| assertEqual 2 (1 + 1)
 
-    ]
+all : Test
+all =
+    describe "TicTacToe Test"
+        [ test "Addition" <|
+            \() ->
+                Expect.equal (3 + 7) 10
```

Comment:
I was having an issue with elm-make. It would flag a whitespace issue with an empty file. I should probably just delete the file entirely until i need it.